manager/container.go

```diff
@@ -244,18 +244,20 @@ func (c *containerData) updateLoad(newLoad uint64) {
 }
 
 func (c *containerData) updateStats() error {
-	stats, err := c.handler.GetStats()
-	if err != nil {
+	stats, statsErr := c.handler.GetStats()
```

Comment:
We should make it explicit in this error message that we went ahead and pushed stats even though an error was encountered.